pkgs/by-name/st/stremio/package.nix

```
@@ -3,39 +3,41 @@
   stdenv,
   fetchFromGitHub,
   fetchurl,
+  libsForQt5,
   ffmpeg,
   mpv,
   nodejs,
-  qmake,
-  qtwebengine,
-  wrapQtAppsHook,
 }:
-
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "stremio-shell";
   version = "4.4.168";
 
   src = fetchFromGitHub {
     owner = "Stremio";
-    repo = pname;
-    rev = "v${version}";
-    fetchSubmodules = true;
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "stremio-shell";
```
using pname is considered an anti pattern